5-Cookies/client-react/src/test/utils.ts

```diff
@@ -0,0 +1,13 @@
+import { v4 as uuidv4 } from "uuid";
+
+export const mockTodo = () => {
```

Comment:
change this function to be a builder of todos. so it'll accept overrides (same as you did in the driver) e.g something like:
```
export const buildTodoItem: Item = (overrides: Partial<item>) => ({
id: chance.guid(), 
text: chance.guid(), 
isFinished: false, 
...overrides 
})
```

Reply:
Ok, I'll will change `mockTodo` function to your suggestion